client/src/components/table/Table.js

```diff
@@ -6,104 +6,152 @@ import 'ag-grid-community/dist/styles/ag-grid.css';
 import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
 
 const Table = () => {
+  // Sort Birth Dates in ascending order (earlier dates first)
+  const dateComparator = (value1, value2) => {
+    const date1 = new Date(value1);
+    const date2 = new Date(value2);
+    return date1 - date2;
+  };
+
+  // Sort Grad Year && Generation in ascending order (earlier cohorts first)
+  const semesterComparator = (value1, value2) => {
+    const sem1 = value1.split(' ')[0];
+    const year1 = value1.split(' ')[1];
+    const sem2 = value2.split(' ')[0];
+    const year2 = value2.split(' ')[1];
+    if (year1 < year2) {
+      return -1;
+    } else if (year1 > year2) {
+      return 1;
+    } else {
+      if (sem1 === 'Fall' && sem2 === 'Spring') {
+        return 1;
+      } else if (sem1 === 'Spring' && sem2 === 'Fall') {
+        return -1;
+      } else {
+        return 0;
+      }
+    }
+  };
+
   const [columnDefs] = useState([
```

Comment:
Also wondering do you know if `columnDefs` has to be state, or can it just be a constant?

Reply:
Ah yes, I kept this here because I thought we might need to limit the visibility of some columns based on role, but I checked the PRD again and it seems like we don't need to

Reply:
I actually think some columns aren't visible depending on level. The way the /api/members endpoint works right now is it doesn't return some columns like areDuesPaid if you're not at director level.

Reply:
Oh right, I think then it doesn't need to be a state variable since we can just set the result of the endpoint directly as the columns